nomic/atlas.py

```diff
@@ -165,12 +166,13 @@ def map_text(
     projection_spread: float = DEFAULT_PROJECTION_SPREAD,
     duplicate_detection: bool = False,
     duplicate_threshold: float = DEFAULT_DUPLICATE_THRESHOLD,
+    upload_batch_size: int = 1000
```

Comment:
This should not be a user-facing parameter. To choose a size anything other than haphazardly, a user would need to know details of our backend ingest system. We shouldn't make users choose parameters they can't set.